tck/bridge-tck-common/src/main/java/com/liferay/faces/bridge/tck/common/util/BridgeTCKUtil.java

```diff
@@ -26,15 +26,13 @@
 public class BridgeTCKUtil {
 
 	public static boolean isHeaderOrRenderPhase(Bridge.PortletPhase portletPhase) {
-		return (Bridge.PortletPhase.HEADER_PHASE.equals(portletPhase) ||
-			Bridge.PortletPhase.RENDER_PHASE.equals(portletPhase));
+		return Bridge.PortletPhase.HEADER_PHASE.equals(portletPhase);
```

Comment:
@stiemannkj1 The return value of this method no longer reflects the "or" condition in the method name. Is the intent here that in Portlet 3.0, that `true` should only be returned for HEADER_PHASE in order to ensure that the bridge is doing things according to Spec?